lib/api/core.dart

```diff
@@ -7,6 +7,14 @@ import '../log.dart';
 import '../model/localizations.dart';
 import 'exception.dart';
 
+/// A fused json + utf8 decoder.
```

Comment:
Prose should use standard orthography:
```suggestion
/// A fused JSON + UTF-8 decoder.
```

even though identifiers often don't.

(Same thing goes for the commit message.)